lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -2837,6 +2836,18 @@ namespace Js
 
         Field(JavascriptFunction*)* builtinFuncs = library->GetBuiltinFunctions();
 
+#ifdef ENABLE_JS_BUILTINS
```

Comment:
One quality of life improvement may be to always have IsJsBuiltInEnabled() a valid function call, and have the ifdef check in there. That way all of these calls don't need their own ifdef.